### ISSUE TITLE
linter: unknown ESLint rules (e.g. custom rules) are reported as unused directives instead of being ignored?

### ISSUE THREAD

**Issue by Boshen** [oxc-project/oxc#11983]
Is it by design or a bug that unknown ESLint rules (e.g. custom rules) are reported as unused directives instead of being ignored?

_Originally posted by @donaldpipowitch in https://github.com/oxc-project/oxc/issues/9223#issuecomment-3018997444_

**Comment by taearls** [comment]
Opened a PR for this issue [here](https://github.com/oxc-project/oxc/pull/14818).

**Comment by donaldpipowitch** [comment]
FYI, interestingly I'd expect to report unknown oxlint rules which doesn't seem to happen right now. Something like `// oxlint-disable-next-line typoscript/no-unnecessary-condition` (see `typoscript`) is not reported by `reportUnusedDisableDirectives: 'error',`

**Comment by camc314** [comment]
Hi, we've taken a slightly different approach here.

You can disable respecting eslint directives with the following config:
```
{
    "options": {
        "respectEslintDisableDirectives": false
    },
```

This should address the concers raised by this issue - please let me know if not.

Thanks!
